content/release/1.4.1.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Release 1.4.1 available
+date: 2024-11-24
+linked: true
+---
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+![Hot-Springs-haiku](releases/1.4.0.jpg)
+
+Apache Ozone 1.4.1 is a minor bug fix release. Compared to Ozone 1.4.0,
+this version resolves some bugs to improve stability and reliability.
```

Comment:
Would be nice to include a link to the list of commits and/or Jira issues resolved in 1.4.1.

- https://github.com/apache/ozone/compare/ozone-1.4.0...ozone-1.4.1 (please push tag `ozone-1.4.1` to make this link work)

- https://issues.apache.org/jira/issues/?jql=project%20%3D%20HDDS%20AND%20fixVersion%20%3D%201.4.1%20AND%20status%20IN%20(Resolved%2C%20Closed)%20ORDER%20BY%20priority%20DESC

Reply:
Tag ozone-1.4.1 has been pushed, I will add this.